CONTRIBUTING.md

```diff
@@ -15,13 +15,13 @@ Provide as much information as you can. The easier it is for us to recreate your
 Feature requests
 ----------------
 
-If you find yourself wishing for a feature that doesn't exist in OpenJFX, you are probably not alone. There are bound to be others out there with similar needs. Many of the features that OpenJFX has today have been added because our users saw the need.
```

Comment:
I think you should restore this sentence. It feels uplifting and encouraging, while the other added sections feel pretty discouraging at first sight for newcomers to the project.

Reply:
Fair point. I don't mind restoring it.

Reply:
OK, I can add this back, although I might reword it slightly.